### DISCUSSION TITLE
Is IP Camera Support going to be implemented ?

### DISCUSSION THREAD

**Discussion by stixoffire** [obsproject/obs-studio#11672] [Development Questions] [answered]
With all of the IP Cameras out there - especially ONVIF variety , and I am sure there will be WiFi and other connectivity , Can we get support to add those types of video sources directly into OBS Studio ?

**Comment by Fenrirthviti** [comment, marked as answer]
"ONVIF" support is unlikely to be something we add natively, but OBS already supports RTSP feeds which is what most IP cameras already use.

We recommend you use a dedicated IP camera management suite if you're using IP cameras and just capture that, as OBS is a compositor, and not typically suited for direct control of such things.
